.travis.yml

```diff
@@ -27,6 +27,6 @@ install:
 # command to run tests
 script:
   - flake8 . --exclude=.venv,.build --ignore=E501,F403,E402
-  - nosetests --with-doctest deeptools/test `ls deeptools/*.py | grep -v correct`
+  - nosetests --with-doctest deeptools/test deeptools --ignore-files="correctGCBias"
```

Comment:
I had to give explicitly `ls deeptools/*.py` because nosetests was not running all doctest by default when simply giving the directory. Does it work for you?